bundles/org.openhab.binding.sonos/src/main/resources/OH-INF/addon/addon.xml

```diff
@@ -6,6 +6,7 @@
 	<type>binding</type>
 	<name>Sonos Binding</name>
 	<description>This is the binding for the Sonos multi-room audio system.</description>
+	<connection>cloud</connection>
```

Comment:
Shouldn't this be:
```suggestion
	<connection>local</connection>
```

?

Reply:
And only now read the note you already put. 😄 I think we can declare this as local, even if some limited functionality extends into the cloud. If you remove the internet connection, this binding will still work.

Reply:
Look at my second message.

Reply:
The description puts us between two choices since it neither requires cloud to run (thus local) or only interacts locally (thus not local):
https://next.openhab.org/docs/developer/addons/addon.html#xml-structure-for-add-on-definitions

I would still opt for local here, but @kaikreuzer / @J-N-K - maybe we already need to further refine these options, or adapt documentation slightly? Hybrid bindings seem not fully covered.

Reply:
The definition is in https://github.com/openhab/openhab-core/issues/2058

> highlight if a binding/automation/... interacts locally or if it connects to an online service (seldom or continuously)

Reply:
Maybe we should create an option that would say that the binding can work without Internet but then with certain degraded features ?

Reply:
> I think we can declare this as local, even if some limited functionality extends into the cloud.

Done

Reply:
Leaving this comment open. We can merge this PR, but we still need to at least clarify the documentation, since Sonos currently doesn't match any of the documented options:
- `local` for add-ons that only interact locally
- `cloud` for add-ons that require a cloud connection
- `cloudDiscovery` for add-ons that require a cloud connection for set-up

We are closest to `local`, but the word "only" disqualifies when reading this strictly.